llvm/include/llvm/SYCLLowerIR/SYCLPropagateAspectsUsage.h

```diff
@@ -17,10 +17,21 @@
 
 namespace llvm {
 
+struct SPAUOptions {
+  SmallVector<StringRef, 8> Targets;
```

Comment:
To avoid ambiguities, a more fitting name would be `TargetFixedAspects`.

Reply:
Done [7043cdc](https://github.com/intel/llvm/pull/7115/commits/7043cdc3454840ab6d7ef5b5f953d8062205b5a1)